storefront/app/helpers/spree/featured_taxon_helper.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module FeaturedTaxonHelper
+    # @param currency [String]
+    # @param featured_taxon [Spree::PageSections::FeaturedTaxon]
+    # @return [ActiveRecord::Relation<Spree::Product>] limited set of products for the featured taxon
+    def taxon_products(currency, featured_taxon)
```

Comment:
```suggestion
    def taxon_products(taxon)
```

1. Let's make this universal and pass taxon only
2. No need to pass currency, we always have access to `current_currency` in views/helpers/controllers - also in the near future, not only will currency affect these product lists, but also user groups, regions and so on